src/components/Profile/index.js

```diff
@@ -90,7 +93,16 @@ const Profile = props => {
         </div>
         <InterestsGrid interestList={userInterests} />
       </div>
-
+      <div className="start-conversation-button">
```

Comment:
This entire div should be wrapped in the conditional rendering clause, since in own profile, empty div is still rendered and looks like a straight gray line in mobile view.

Reply:
Wrapped :)